.github/workflows/docs.yaml

```
@@ -6,16 +6,49 @@ on:
       - created
   push:
     branches: [ "main" ]
-  # Allow running this workflow manually from the Actions tab
+    
+  pull_request:
+    branches: [ "main" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)